plugins/org.eclipse.xtext/src/org/eclipse/xtext/formatting2/regionaccess/ITextRegionAccess.java

```diff
@@ -63,15 +73,15 @@
 	ITextSegment expandRegionsByLines(int leadingLines, int trailingLines, ITextSegment... regions);
 
 	/**
-	 * @return The very first {@link IHiddenRegion} inside the linked list of alternating {@link IHiddenRegion hidden regions} and
-	 *         {@link ISemanticRegion semantic regions}.
+	 * @return The very first {@link IHiddenRegion} inside the linked list of alternating {@link IHiddenRegion hidden
+	 *         regions} and {@link ISemanticRegion semantic regions}.
 	 */
 	IHiddenRegion getFirstRegionInFile();
 
 	/**
 	 * @return the {@link RuleCall} or the assigned {@link Action} that led to the construction of this EObject.
```

Comment:
Is this JavaDoc now obsolete / wrong? Otherwise we should have sticked to the more concrete return type